server/verify_test.go

```diff
@@ -1140,10 +1161,9 @@ func TestVerifyAttestationWithTdx(t *testing.T) {
 	}
 	defer ak.Close()
 
-	nonce := []byte("super secret nonce")
```

Comment:
We should use a different value for the TPM nonce. That way we make sure the TPM and TEE nonces don't subtly depend on each other in attest. So, make a `tpmNonce` and a `teeNonce`. This makes the tests easier to read.

Reply:
Done